docs/tasks/run_jobs.md

```diff
@@ -0,0 +1,190 @@
+- You should include the resource requests for each Job Pod.
+
+Here is a sample Job with three Pods that just sleep for a few seconds.
+This sample is also available in [config/samples/sample-job.yaml](/config/samples/sample-job.yaml).
+
+```yaml
+# sample-job.yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: sample-job-
+  annotations:
+    kueue.x-k8s.io/queue-name: main
+spec:
+  parallelism: 3
+  completions: 3
+  suspend: true
+  template:
+    spec:
+      containers:
+      - name: dummy-job
+        image: gcr.io/k8s-staging-perf-tests/sleep:latest
+        args: ["30s"]
+        resources:
+          requests:
+            cpu: 1
+            memory: "200Mi"
+      restartPolicy: Never
+```
+
+## 2. Run the Job
+
+You can run the Job with the following command:
+
+```shell
+kubectl create -f sample-job.yaml
+```
+
+Internally, Kueue will create a corresponding [QueuedWorkload](/docs/concepts/queued_workload.md)
+for this Job with a matching name.
+
+```shell
+kubectl -n default get workloads
+```
+
+The output will be similar to:
+
+```
+NAME               QUEUE   ADMITTED BY     AGE
+sample-job-sl4bm   main                    1s
+```
+
+## 3. (Optional) Monitor the status of the workload
+
+You can run the following command to see the status of the workload.
+
+```shell
+kubectl -n default describe workload sample-job-sl4bm
+```
+
+If the ClusterQueue doesn't have enough quota to run the workload, the output
+will be similar to:
+
+```
+Name:         sample-job-jtz5f                                                                                                                                       
+Namespace:    default                                                                                                                                                
+Labels:       <none>                                                                                                                                                 
+Annotations:  <none>                                                                                                                                                 
+API Version:  kueue.x-k8s.io/v1alpha1                                                                                                                                
+Kind:         QueuedWorkload                                                                                                                                         
+Metadata:
+  ...
+Spec:
+  ...
+Status:
+  Conditions:
+    Last Probe Time:       2022-03-28T19:43:03Z
+    Last Transition Time:  2022-03-28T19:43:03Z
+    Message:               workload didn't fit
+    Reason:                Pending
+    Status:                False
+    Type:                  Admitted
+Events:               <none>
+```
+
+After some time, when the ClusterQueue has enough quota to run the workload,
+the output of `kubectl -n default get workloads` will look similar to:
```

Comment:
When the ClusterQueue has enough quota, the cluster will run the workload.

To see active workloads, run the following command
 `kubectl -n default get workloads` 

The output is similar to the following: